src/View/mainpage/WindowingSlider.py

```diff
@@ -251,9 +276,17 @@ def mouse_release(self, event):
             False,
             False,
             False]
-        level = 100
-        window = 100
-        # windowing_model_direct(level, window, send)
+
+        top_bar = self.index_to_window(self.top)
+        bottom_bar = self.index_to_window(self.bottom)
+
+        level = (top_bar + bottom_bar) * 0.5
+        window = 2 * (bottom_bar - level)
+        level = level - 1000
```

Comment:
magic number... please use a constant or variable with meaningful name.
It's not clear 1000 is the value needed